great_expectations/core/usage_statistics/anonymizers/validation_operator_anonymizer.py

```diff
@@ -23,16 +23,17 @@ def __init__(self, salt=None):
     def anonymize_validation_operator_info(
         self, validation_operator_name, validation_operator_obj
     ):
-        anonymized_info_dict = {}
-        anonymized_info_dict["anonymized_name"] = self.anonymize(
-            validation_operator_name
-        )
-        actions_dict = validation_operator_obj.actions
+        anonymized_info_dict: dict = {
```

Comment:
Why type hint here, but not on my comment above? What is our convention regarding type hints in function bodies?

Reply:
@NathanFarmer If I am not mistaken, in your case there was one declared before.  Here, it is added, because there was none before.

Reply:
I understand, thank you 🙇🏻